cfripper/__version__.py

```diff
@@ -1,3 +1,3 @@
-VERSION = (1, 6, 0)
+VERSION = (1, 7, 0)
```

Comment:
Everything looks good to me. This change could break custom rules though, so it is not backwards compatible, could you confirm? If so it should change the major version.

Reply:
Custom rules can still report/add failures without adding this new parameter. But the failure will include now `resource_types=None` by default.
Which is what I discovered by running and updating the tests.
Happy to make it v2.0.0 instead if we want to be sure about it.

Reply:
Cool, so it is not a breaking change. Will approve